src/App.jsx

```diff
@@ -1,44 +1,32 @@
-import React from 'react';
+import React, { useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 import './App.scss';
+import { Tabs } from './components/Tabs/Tabs';
```

Comment:
Improve imports order 

Useful link:
https://levelup.gitconnected.com/how-to-sort-imports-in-react-project-550f5ce70cbf

Reply:
Better
```
import React, { useState } from 'react';

import 'bulma/css/bulma.css';
import '@fortawesome/fontawesome-free/css/all.css';

import { Tabs } from './components/Tabs/Tabs';

import './App.scss';
```